packages/server/src/entities/comment/mutations/commentDelete.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  GraphQLID,
+  GraphQLString,
+  GraphQLNonNull
+} from 'graphql'
+import { CommentModel } from '../commentModel'
+import { fromGlobalId, mutationWithClientMutationId } from 'graphql-relay'
+import { getHeadersPayload } from '../../../auth/getHeadersPayload'
+import { errorField } from '../../../graphql/errorField'
+
+export const commentDelete = mutationWithClientMutationId({
+  name: 'commentDelete',
+  description: 'Delete a comment using its id',
+  inputFields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLID)
+    }
+  },
+  outputFields: {
+    id: {
+      type: GraphQLString,
+      description: 'The id of the deleted comment',
+      resolve: response => response.id
+    },
+    ...errorField
+  },
+  mutateAndGetPayload: async ({ ...comment }, ctx) => {
+    const { error, payload } = getHeadersPayload(ctx)
```

Comment:
why not to resolve this into the root o koa?

Reply:
you're right, I hadn't thought about it